### PR TITLE
Change PR template to say fixes issues

### PR DESCRIPTION
**Pull request by BSFishy**
### Description
Change the PR template to include "Fixes" in the `Issues Resolved` section, to make it clearer to contributors on the proper format for the section. This will help maintainers by automatically closing related issues so maintainers don't have to do it manually

### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] All tests pass
  - [x] `yarn lint`
  - [x] `yarn test-unit`
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ### Description
 [Describe what this change achieves]
- 
+
```
Comment (BSFishy) [inline .github/pull_request_template.md:3]: Sorry to reviewers for including these changes to remove the space on empty lines but it's been getting on my nerves for a while now and I don't want to waste the time making a new PR right now

**Review by joshuarrrr — APPROVED** [review]
Thanks, you beat me to this! Let's see if this helps with linking more issues and PRs.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ### Description
 [Describe what this change achieves]
```
Comment (ashwin-pc) [inline .github/pull_request_template.md:2]:
```suggestion
<!-- Describe what this change achieves -->
```
Makes this a comment that does not show up in the PR descriptions

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
- 
+
 ### Issues Resolved
-[List any issues this PR will resolve]
- 
+Fixes [issues this PR will resolve]
```
Comment (ashwin-pc) [inline .github/pull_request_template.md:5]:
```suggestion
<!-- List any issues this PR will resolve. -->
<!-- Example: Fixes #1234 -->
```

**Comment by joshuarrrr** [comment]
@BSFishy I set the DCO check to pass, but you may want to check your GitHub settings. Those commits were signed with provomat@amazon.com, but the DCO check fails because that doesn't match you default GitHub email address.

**Review by joshuarrrr — APPROVED** [review]

**Review by AMoo-Miki — APPROVED** [review]

**Pull request merged**